src/config/env/heroku.json

```diff
@@ -35,5 +35,6 @@
   "GATSBY_VOTING_POWER_TO_PASS_GOVERNANCE": "500",
   "GATSBY_VOTING_POWER_TO_PASS_POI": "10000",
   "GATSBY_VOTING_POWER_TO_PASS_POLL": "500",
-  "GATSBY_GRANT_SIZE_MINIMUM": "1"
+  "GATSBY_GRANT_SIZE_MINIMUM": "1",
+  "DISCORD_SERVICE_ENABLED": "false"
```

Comment:
This variable is not necessary here as the discord service is only used server side 🤔